backend/Dockerfile

```diff
@@ -16,5 +16,17 @@ COPY --from=builder /app ./
 RUN apk upgrade --update-cache --available && \
     apk add openssl && \
     rm -rf /var/cache/apk/*
+
+RUN apk add --no-cache \
+    chromium \
+    nss \
+    freetype \
+    harfbuzz \
+    ca-certificates \
+    ttf-freefont
+
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
+ENV PUPPETEER_EXECUTABLE_PATH /usr/bin/chromium-browser
+
 EXPOSE 3000
-CMD ["npm", "run", "start:prod"]
+CMD ["npm", "run", "start:prod"]
```

Comment:
Please add [newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) below here. You can configure this option in your IDE.

Reply:
I checked the POSIX rule and updated my IDE settings to automatically add newlines in the future. Thank you.